modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -10423,6 +10423,7 @@ object-definition-table-name=Table Name
 object-entry-value-exceeds-integer-field-allowed-size=Object entry value exceeds integer field allowed size.
 object-entry-value-exceeds-long-field-allowed-size=Object entry value exceeds long field allowed size.
 object-entry-value-exceeds-maximum-long-field-allowed-size=Object entry value exceeds maximum long field allowed size.
+object-entry-value-exceeds-the-maximum-length-of-x-characters-for-object-field-x=Object entry value exceeds the maximum length of {0} characters for object field {1}.
```

Comment:
Did you validate this message with design/product?

Reply:
I changed the message after validating with design/product